lib/mappers/__tests__/__fixtures__/mapper.sketch.js

```diff
@@ -12,7 +12,7 @@ exports.DOCUMENT_COLORS = sampleOutput.document.assets.colorAssets
   .map(({ color, _class, name }) => ({ ...color, _class, name }));
 exports.DOCUMENT_GRADIENTS = sampleOutput.document.assets.gradientAssets
   .map(({ gradient, _class }) => ({ ...gradient, _class }));;
-exports.TEXT_STYLES = sampleOutput.document.layerTextStyles;
+exports.TEXT_STYLES = sampleOutput.document.layerTextStyles.objects;
```

Comment:
wow, did the Sketch API change in the mean time? Make sure that this still yields the same result as it did before in outputs.

Reply:
This is a red flag to me when I check the diff for sketch/sample_dump.json

> File renamed without changes.

If it didn't change this will prob. be a regression since you are accessing a layer deeper.